src/python/deepgnn/graph_engine/snark/decoders.py

```diff
@@ -436,7 +513,7 @@ def decode(self, line: str) -> Iterator[Tuple[int, int, int, float, list]]:
         node_weight = float(columns[2]) if columns[2] else 0.0
         yield node_id, -1, node_type, node_weight, self._parse_feature_string(
             columns[3]
-        )
+        ), None, None
```

Comment:
I think we should remove the ", None, None" since these are supposed to be optional.

Reply:
Optional type means the value can be `None`, not that it can be ommited from result list.